components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -499,9 +502,9 @@ public User updateUser(User user, Map<String, Boolean> requiredAttributes) throw
     private List<Object> filterUsers(Node node, Map<String, Boolean> requiredAttributes, int offset, int limit,
             String sortBy, String sortOrder, String domainName) throws CharonException {
 
-        //Handle single attribute search
+        // Handle single attribute search.
         if (node instanceof ExpressionNode) {
-            return getFilteredUsersBySingleAttribute((ExpressionNode) node, requiredAttributes, offset, limit, sortBy,
+            return filteredUsersBySingleAttribute((ExpressionNode) node, requiredAttributes, offset, limit, sortBy,
                     sortOrder, domainName);
 
```

Comment:
Shall we remove this new line?

Reply:
fixed:  79e0f61a639ffe10869df3f4c7d5bf5611e2b731